src/MustardBlack.Assets.Build/Program.cs

```diff
@@ -71,21 +72,29 @@ static string ProcessJs(string options, IAssetLoader assetLoader, string path)
 		{
 			var opts = options.Split(':');
 
-			var assets = assetLoader.GetAssets(path, AreaJavascriptHandler.FileMatch);
+			var assets = assetLoader.GetAssets(path, AreaJavascriptHandler.FileMatch).ToArray();
 			if (!assets.Any())
 				return null;
 
 			string result;
-
 			IJavascriptPreprocessor javascriptPreprocessor;
+			

```

Comment:
if babel
if minify
if nuglify

can it contain babel and nuglify at the same time?

Reply:
Yes. You just pass `babel:minify:nuglify` as a parameter to the exe. The value passed is configured at the project level.